partcad/src/partcad/builtin/render/render_dxf.py

```
@@ -0,0 +1,74 @@
+            elif segment.__class__.__name__ in ["CubicBezier", "QuadraticBezier", "Arc"]:
+                for i in range(CURVE_SEGMENTS):
+                    p0 = segment.point(i / float(CURVE_SEGMENTS))
+                    p1 = segment.point((i + 1) / float(CURVE_SEGMENTS))
+                    msp.add_line(Vec2(p0.real, p0.imag), Vec2(p1.real, p1.imag))
+
+    doc.saveas(dxf_file)
+
+
+def process(path, request):
+    try:
+        svg_path = tempfile.mktemp(".svg")
```

Ruff (0.16.1)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)